README mwmichelpach1.md

```diff
@@ -1,5 +1,12 @@
 # Cloudera Director Public Scripts
 
+## Contributing Code
+
+In order to contribute code you must sign one of two agreements. First if you are choosing to contrubute on your own behalf then sign the "ICLA agreement (https://github.com/cloudera/livy/wiki/Individual-Contributor-License-Agreement-(ICLA)". If you are contrbuting on the behalf of an organization or a company you must sign the the "CCLA agreement (https://github.com/cloudera/livy/wiki/Corporate-Contributor-License-Agreement-(CCLA)". 
```

Comment:
Remove "First".
No need for quotes around "ICLA agreement". Same for CCLA.
Typo in contrubute.
No need for "the" in "on the behalf"
There's an extra "the" before CCLA.